### ISSUE TITLE
Can we have a json template feature just like we have for yaml template. 

### ISSUE THREAD

**Issue by Ss081994** [terraform-google-modules/terraform-google-slo#103]
### TL;DR

The json template is readily available to download from the console. The json template is natively supported and can be a feature in this module.

### Detailed design

A sample json for cloud function execution: 

{
  "displayName": "90% - Distribution Cut - Rolling 7 days",
  "goal": 0.9,
  "rollingPeriod": "604800s",
  "serviceLevelIndicator": {
    "requestBased": {
      "distributionCut": {
        "distributionFilter": "metric.type=\"cloudfunctions.googleapis.com/function/execution_times\" resource.type=\"cloud_function\"",
        "range": {
          "min": null,
          "max": 10
        }
      }
    }
  }
}
